modules/distr-tracing-tempo-object-storage-setup-aws-sts-install.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+//* observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="distr-tracing-tempo-object-storage-setup-aws-sts-install_{context}"]
+= Setting up the Amazon S3 storage with the Security Token Service
+
+You can set up the Amazon S3 storage with the Security Token Service (STS) by using the AWS Command Line Interface (AWS CLI).
+
+:FeatureName: The Amazon S3 storage with the Security Token Service
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+.Prerequisites
+
+* You have installed the latest version of the AWS CLI.
+
+.Procedure
+
+. Create an AWS S3 bucket.
+
+. Create the following `trust.json` file for the AWS IAM policy that will set up a trust relationship for the AWS IAM role, which you will create in the next step, with the service account of the TempoStack instance, which you intend to deploy:
```

Comment:
This sentence is long, so I would try to break it in two. Maybe something like this:

Create the `trust.json` file for the AWS IAM policy to set up a trust relationship for the AWS IAM role. This role will be associated with the service account of the TempoStack instance you intend to deploy.

Reply:
Thank you. I've shortened this sentence somewhat. Unfortunately, I tried and tried to break this sentence into two but could not without losing the intended meaning.